src/win.rs

```diff
@@ -161,6 +161,15 @@ fn test_conout() {
     assert!(conout().is_ok())
 }
 
+#[rustversion::before(1.36)]
+unsafe fn uninitialized<T>() -> T {
+    ::std::mem::uninitialized()
+}
+#[rustversion::since(1.36)]
+unsafe fn uninitialized<T>() -> T {
+    ::std::mem::MaybeUninit::uninit().assume_init()
```

Comment:
This is undefined behavior. `assume_init` can't be called until we actually initialize it with `GetConsoleScreenBufferInfo`.

Reply:
I thought an uninitialized value was ok since the value will be initialized by the function soon. But as you pointed, returning uninitialized buffer may cause undefined behavior and we should eliminate it.

<!--
How about initializing the buffer with zero?

```rust
#[rustversion::since(1.36)]
unsafe fn uninitialized<T>() -> T {
    ::std::mem::MaybeUninit::zeroed().assume_init()
}
```

Since every bit is initialized to zero, the behavior won't be undefined. Though I read a documentation of [the buffer info struct](https://docs.microsoft.com/ja-jp/windows/console/console-screen-buffer-info-str), I'm not sure filling with zero is sufficient.-->